patroni/postgresql.py

```diff
@@ -352,15 +372,21 @@ def sysid(self):
             self._sysid = data.get('Database system identifier', "")
         return self._sysid
 
-    def get_local_address(self):
+    def _get_unix_local_address(self):
+        if 'unix_socket_directories' in self._server_parameters:
```

Comment:
because of the indentation it took me a few passes to see how this function can return None. I'd write it as:
if 'unix_socket_directories not in self._server_parameters:
    return None
<the rest of the function>